core/docz-core/src/commands/serve.ts

```diff
@@ -0,0 +1,11 @@
+import spawn from 'cross-spawn'
+import { Arguments } from 'yargs'
+import { parseConfig } from '../config/docz'
+import * as paths from '../config/paths'
+
+export const serve = async (args: Arguments<any>) => {
+  const config = await parseConfig(args)
+  const dist = paths.getDist(config.dest)
```

Comment:
I believe that `cross-spawn` handles relative paths well (that means I could have used `config.dest` directly, but I decided to take a more secure approach.